nixos/modules/services/web-apps/shiori.nix

```diff
@@ -56,9 +76,14 @@ in {
 
           # For SSL certificates, and the resolv.conf
           "/etc"
-        ];
+        ] ++
+        lib.optional (lib.strings.hasInfix "postgres" cfg.databaseUrl)
+          "/run/postgresql" ++
+        lib.optional (lib.strings.hasInfix "mysql" cfg.databaseUrl)
+          "/var/run/mysqld";
```

Comment:
I think there is a case to be made for instances that use external databases here, but I don't have any clever solutions at the moment. Maybe append `&& config.services.{postgresql,mysql}.enable`? Just to be sure - this does not fail if these paths does not exist, right?

Reply:
Good suggestion! implemented that check and worked like a charm! it's good to be extra secure.